examples/plot_DTM_classification.py

```diff
@@ -0,0 +1,80 @@
+"""Distance to trimmed means (DTM) classification.
```

Comment:
I do not think that an separate gallery example for each classification method is THAT useful, unless it has lots of configuration. I think that is better to do a example for each family, such as an example comparing the different depth-based classifiers, and explaining the theory behind each of them and illustrating their differences. Otherwise, this is redundant with the docstring examples.

Reply:
Makes sense! I will work on a unified example once I have DDClassifier since it will be interesting to show, amongst other things, that MaximumDepthClassifier is just a particular case.